test/widgets/message_list_test.dart

```diff
@@ -981,4 +981,70 @@ void main() {
       });
     });
   });
+
+  group('TypingStatus', () {
+    final users = [eg.selfUser, eg.otherUser, eg.thirdUser, eg.fourthUser];
+    final finder = find.descendant(
+      of: find.byType(TypingStatusWidget),
+      matching: find.byType(Text)
+    );
+
+    checkTyping(WidgetTester tester, TypingEvent event, {required String expected}) async {
+      await store.handleEvent(event);
+      await tester.pump();
+      check(finder.evaluate()).single.has((x) => x.widget, 'widget').isA<Text>()
+        .data.equals(expected);
+    }
+
+    final dmMessage = eg.dmMessage(from: eg.otherUser, to: [eg.selfUser]);
+    final dmNarrow = DmNarrow.withUsers(
+      users.map((u) => u.userId).toList(),
+      selfUserId: eg.selfUser.userId);
```

Comment:
This message doesn't appear in this narrow. For that, the `to` argument should have the other users. (The code below assumes the message is in the narrow, so this affects the realism of the test data.)

Then once you do that, there's a handy constructor `DmNarrow.ofMessage`.